examples/animation-controls/animation-controls.js

```diff
@@ -10,14 +10,18 @@ const animationNames = {
 
 updateAnimationMixer = () => {
 
-  const data = {}
+  const data = {useRegExp: true}
   data.clip = 'none'
   Object.entries(animationNames).forEach((name) => {
 
-    const el = document.getElementById(name[0])
-    
+    const regExpEscape = (s) => {
+      return s.replace(/[|\\{}()[\]^$+*?.]/g, '\\$&');
+    }
+
+    el = document.getElementById(name[0])
     if (el.checked) {
-      data.clip = name[1]
+      if (data.clip) data.clip += "|"
```

Comment:
Are you sure you want to set `data.clip = 'none'` initially?
It will generate "none|TRex_Walk" right?
I don't quite understand what your regExpEscape does.

Reply:
Hi, sorry I didn't reply!  I wrote the regexp almost a year ago, and when I came back to it I couldn't immediately remember what I'd been thinking... then other things ame up...

I think you have unerstood the intent here.